comment/views/comments.py

```diff
@@ -69,23 +69,21 @@ def form_valid(self, form):
 
         if settings.COMMENT_ALLOW_ANONYMOUS and not user:
             # send response, please verify your email to post this comment.
-            response_msg = process_anonymous_commenting(self.request, comment)
+            response_msg = process_anonymous_commenting(self.request, _comment)
             messages.info(self.request, response_msg)
         else:
-            comment.save()
-            self.comment = comment
+            _comment.save()
+            self.comment = _comment
 
         return self.render_to_response(self.get_context_data())
 
 
-class UpdateComment(CommentUpdateMixin, BaseCommentView):
+class UpdateComment(CanEditMixin, BaseCommentView):
     comment = None
 
-    def dispatch(self, request, *args, **kwargs):
+    def get_object(self):
         self.comment = get_object_or_404(Comment, pk=self.kwargs.get('pk'))
-        if request.user != self.comment.user:
-            raise PermissionDenied
-        return super().dispatch(request, *args, **kwargs)
+        return self.comment
```

Comment:
This may be replaced with the [`SingleObjectMixin`](https://docs.djangoproject.com/en/dev/ref/class-based-views/mixins-single-object/#single-object-mixins)

Reply:
For now we don't need to bring a cow for glass of milk :D this might be refactor later